docs/handbook/getting-started.md

```diff
@@ -1937,8 +1959,9 @@ Nearly all cards can use the VESA driver, and are therefore supported for 2D.
 
 ### Video card support (3D)
 
-* Beginning with the July 2016 experimental release, both Nvidia and Intel GPU's are now supported.
-* AMD GPU's are not currently supported (VESA driver only).
+* Nearly all NVidia GPU's are supported by the Nvidia binary driver.
+* Beginning with the July 2016 experimental release, most Intel GPU's are now supported.
+* While AMD GPU's are not currently supported (VESA driver only), development is currently underway to provide 3D support for both legacy and modern AMD GPU's.
```

Comment:
Need small correction. DRI on Radeon is not supported, not Radeon completely. The work is going to support 2d with Radeon on newer Xorg. AFAIK, there's no any work in progress to support 3D on Radeon.